spec/pipelines/configuration_spec.rb

```diff
@@ -131,5 +133,29 @@
     it 'will raise an error if it is not a valid version' do
       expect { configuration.pipeline_c.sample_sheet.by_version('v30') }.to raise_error(Version::Error, 'Not a valid version')
     end
+
+    it 'will be able to access fields for a specific version' do
+      expect(configuration.pipeline_c.sample_sheet.by_version('v10').field_10).to eq('b')
+    end
+
+    it 'will not be able to access fields from another version' do
+      assert_raises NoMethodError do
```

Comment:
`expect { configuration.pipeline_c.sample_sheet.by_version('v10').field_20 }.to raise_error(NoMethodError)`
Have you tried the above? I don't think i've come across assert_raises in rspec before. Feels more like a pytest style assertion.